examples/mnist/simple_model_training.py

```diff
@@ -27,18 +28,24 @@
                     logging_level=logging.INFO)
 
 
-# initiate model training
-(
-    train_stats,  #training statistics
-    _,
-    output_directory  # location for training results saved to disk
-) = model.train(
-    training_set='./data/mnist_dataset_training.csv',
-    test_set='./data/mnist_dataset_testing.csv',
-    experiment_name='simple_image_experiment',
-    model_name='single_model',
-    skip_save_processed_input=True
-)
+class MnistDataset(Mnist):
```

Comment:
We should be able to simplify this to:

```
from ludwig.datasets import mnist

...

# load and split MNIST dataset
dataset_df = mnist.load()
training_set = dataset_df[dataset_df[SPLIT] == "0"]
test_set = dataset_df[dataset_df[SPLIT] == "2"]

# initiate model training
(
    train_stats,  #training statistics
    _,
    output_directory  # location for training results saved to disk
) = model.train(
    training_set=training_set,
    test_set=test_set,
    experiment_name='simple_image_experiment',
    model_name='single_model',
    skip_save_processed_input=True
)
```

We should also do the same in `advanced_model_training.py`.

Once that's done, we should be good to land this.

Reply:
Done, I also added to the advanced_model_training, see latest commit